src/apps/js_v8/v8_runner.h

```diff
@@ -94,7 +94,7 @@ namespace ccf
     {
       return isolate_;
     }
-    operator v8::Isolate *()
+    operator v8::Isolate*()
```

Comment:
Can you run the ci-checks script with -f to fix the formatting?

`./scripts/ci-checks.sh -f`

This will automatically fix the formatting, and the CI will pass.

Reply:
When I run that I get no changes.

Reply:
Manually changed back.